cpp_tests/BUILD

```diff
@@ -27,14 +27,22 @@ tfrt_cc_library(
     visibility = [
         "//visibility:public",
     ],
+    deps = [
+        "@clink//:clink_kernels_alwayslink",
```

Comment:
Ideally we want to have a guidance on which dependencies should be put in `common` here. For example, it seems that `@clink//:clink_kernels` can also be moved here.